server.go

```diff
@@ -12,8 +12,14 @@ import (
 	"time"
 )
 
-// Maximum number of TCP queries before we close the socket.
-const maxTCPQueries = 128
+const (
+	// Maximum number of TCP queries before we close the socket.
+	maxTCPQueries = 128
+	// Maximum number of incoming DNS messages in queue.
+	maxQueueSize = 1000000
+	// Maximum number of workers.
+	maxWorkers = 100
```

Comment:
Why a 100?

Reply:
Observed that 100 is optimal value, <100 and >100 drops performance.

Reply:
But probably we need more tests.